### PR TITLE
add VS Code Tunnel recommendations

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

File: mkdocs/docs/HPC/VScode.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# VS Code Tunnel
+
+Please keep in mind, that this is not a VS Code manual,
```
Comment (stdweird) [inline mkdocs/docs/HPC/VScode.md:3]: use "`VS Code`" (with backticks, and consistent uppercase, see line below)

File: mkdocs/docs/HPC/VScode.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+it is only our recommendation how to connect to the {{ hpcinfra }} using the VS code tunnel. 
+
+For VS Code documentation, see <https://code.visualstudio.com/docs>
+
+We only support VS Code tunnel access via the VS Code application (installed locally),
+since web access might not work from all browser or operational system.
+The only confirmed way to connect via a browser is using Windows 11 and Edge.
```
Comment (stdweird) [inline mkdocs/docs/HPC/VScode.md:10]: This neds to be clarified: this is the to connect to the tunnel URL via a browser

File: mkdocs/docs/HPC/VScode.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+For VS Code documentation, see <https://code.visualstudio.com/docs>
+
+We only support VS Code tunnel access via the VS Code application (installed locally),
+since web access might not work from all browser or operational system.
+The only confirmed way to connect via a browser is using Windows 11 and Edge. 
+
+## Prerequisites
+
+We do recommend to install remote developent for the VS Code App.
```
Comment (stdweird) [inline mkdocs/docs/HPC/VScode.md:14]: `development` (but i don't know what this sentence means)

File: mkdocs/docs/HPC/VScode.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+In the [web portal](./web_portal.md), under the "Interactive Apps" choose VS Code Tunnel,
+select the parameters and launch your job by clicking ++"Launch"++
+When your job has started, connect by clicking ++"Connect"++
+
+In the new browser tab or window, for the question
+`How would you like to log in to Visual Studio Code?` choose `Microsoft Account` 
+(using the arrow keys on your keyboard, then hit ++"Enter"++).
+Then follow the instruction to authenticate yourself.
+If you get an error message something like "You do not have access to this"
+durng the Microsoft Authentication, then you might have to
```
Comment (stdweird) [inline mkdocs/docs/HPC/VScode.md:28]: during

File: mkdocs/docs/HPC/VScode.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+In the [web portal](./web_portal.md), under the "Interactive Apps" choose VS Code Tunnel,
+select the parameters and launch your job by clicking ++"Launch"++
+When your job has started, connect by clicking ++"Connect"++
+
+In the new browser tab or window, for the question
+`How would you like to log in to Visual Studio Code?` choose `Microsoft Account` 
+(using the arrow keys on your keyboard, then hit ++"Enter"++).
+Then follow the instruction to authenticate yourself.
+If you get an error message something like "You do not have access to this"
```
Comment (stdweird) [inline mkdocs/docs/HPC/VScode.md:27]: is there a way to make this a warn or tip or info block? @boegel  do you know how to do this?

File: mkdocs/docs/HPC/VScode.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+durng the Microsoft Authentication, then you might have to
+[whitelist yourself](https://dictselfservice.ugent.be/index.php?page=requestform&form=deviceCodeAuth)
+(You have to do it only once.)
```
Comment (stdweird) [inline mkdocs/docs/HPC/VScode.md:30]: add that this is only for ugent users and they can use `Need VS Code tunnel for HPC` as motivation text

File: mkdocs/docs/HPC/VScode.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+If you are not connected automatically to the tunnel, please select "connect to..." and then
+"Connet to Tunnel" using Microsoft Account and select the tunnel named `vsc-<your vsc account>-<cluster>`.
```
Comment (stdweird) [inline mkdocs/docs/HPC/VScode.md:33]: connect

File: mkdocs/docs/HPC/VScode.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+Please note that right now you can only have one tunnel per cluster.
+
+If you are asked to update VS Code CLI on your remote server, choose ++"Not Now"++.
+You do not have permission to update the CLI, as it is centrally installed.
+This means that your VS Code App is newer than the CLI on our systems.
+We update the centrally installed CLI time to time,
+but the upgrade rate of the App/CLI is very high,
+so it is hard for us to follow it tightly.
```
Comment (stdweird) [inline mkdocs/docs/HPC/VScode.md:41]: `Feel free to point newer versions out to us using the software install form` ?

File: mkdocs/docs/HPC/VScode.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+
+Please do not use VS Code to ssh to the login nodes,
+but always use a VS Code Tunnel conection,
```
Comment (stdweird) [inline mkdocs/docs/HPC/VScode.md:44]: connection

File: mkdocs/docs/HPC/VScode.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+peferably using the [interactive and debug cluster](./interactive_debug.md)
```
Comment (stdweird) [inline mkdocs/docs/HPC/VScode.md:45]: preferably

File: mkdocs/docs/HPC/VScode.md (hunk 1 above)
Comment (stdweird) [reply mkdocs/docs/HPC/VScode.md:3]: maybe rename the md file to vscodetunnel, i think the url now would be like `.../vscode`. if this is only about the tunnel, make it more clear

File: mkdocs/docs/HPC/VScode.md [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+In the new browser tab or window, for the question
+`How would you like to log in to Visual Studio Code?` choose `Microsoft Account` 
+(using the arrow keys on your keyboard, then hit ++"Enter"++).
+Then follow the instruction to authenticate yourself.
+If you get an error message something like "You do not have access to this"
+durng the Microsoft Authentication, then you might have to
+[whitelist yourself](https://dictselfservice.ugent.be/index.php?page=requestform&form=deviceCodeAuth)
+(You have to do it only once.)
+
+If you are not connected automatically to the tunnel, please select "connect to..." and then
```
[the remark this answers is not in the document]
Comment (hajgato) [reply mkdocs/docs/HPC/VScode.md:32]: Yes. When you open `VS Code` App it will try to connect to the tunnels that were open when you closed the app.

**Pull request merged**